scripts/tools/GlobalTimestamps.yml

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Tool Experimental Extension for Global Metrics Timestamps"
+version: "1.5"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "Global Metrics Timestamps Experimental Extension Name"
+version: "1.5"
+name: $T_GLOBAL_METRICS_TIMESTAMPS_EXP_NAME
+value: '"$XT_experimental_global_metrics_timestamps"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Global Metrics Timestamps Experimental Extension Version(s)"
+version: "1.5"
+name: $x_global_metrics_timestamps_exp_version_t
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Returns global device timestamps synchronized with metrics timesstamps, optionally mapped to host."
+class: $tMetricGroup
+name: GetTimestampsExp
```

Comment:
will the final name be `zetDeviceGetMetricTimestamps` ?

Reply:
@matcabral : that's one option. if you have another option, could you propose?

Reply:
that name is good, my point was about how yaml woudl concatenate strings.

Reply:
@matcabral  final name would be zetMetricGroupGetTimestampsExp